dev/ci/user-overlays/669-maximedenes-ssr-merge.sh

```diff
@@ -0,0 +1,4 @@
+if [ $TRAVIS_PULL_REQUEST == "669" ] || [ $TRAVIS_BRANCH == "ssr-merge" ]; then
+    mathcomp_CI_BRANCH=ssr-merge
+    mathcomp_CI_GITURL=https://github.com/maximedenes/math-comp.git
+fi
```

Comment:
Please make the PR number a 4 or 5 digits number so they are properly shorted by shells; we will surpass the 1000 PR soon